pkg/operator/controller/nodeobservability/scc.go

```diff
@@ -66,39 +71,42 @@ func (r *NodeObservabilityReconciler) desiredSecurityContextConstraints(nodeObs
 	var priority int32 = 10
 
 	scc := &securityv1.SecurityContextConstraints{
-		ObjectMeta: metav1.ObjectMeta{
-			Name: sccName,
-		},
+		ObjectMeta:               metav1.ObjectMeta{Name: sccName},
+		Priority:                 &priority,
 		AllowPrivilegedContainer: true,
-		AllowHostIPC:             false,
+		DefaultAddCapabilities:   nil,
+		RequiredDropCapabilities: []corev1.Capability{"MKNOD"},
+		AllowedCapabilities:      nil,
+		AllowHostDirVolumePlugin: true,
+		Volumes:                  []securityv1.FSType{securityv1.FSTypeHostPath, securityv1.FSTypeSecret, securityv1.FSTypeConfigMap},
 		AllowHostNetwork:         true,
```

Comment:
```suggestion
		AllowHostNetwork:         false,
```
We don't use hostNetwork for now.